### PR TITLE
Comp/modal

### PR DESCRIPTION
**Pull request by manish591**
Added Modal component.

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Looks good to merge, Manish!

File: pages/components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+.dialog--hide {
+    display: none;
+}
+
+.dialog {
+    position: fixed;
+    inset-block-start: 30%;
+    inset-inline-start: 50%;
+    transform: translateX(-50%);
+    inline-size: 250px;
```
Comment (ramalingaa) [inline pages/components/modal/modal.css:27]: try not using absolute units

File: pages/components/modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+    padding-inline: 1rem;
+}
+
+.dialog__cta--confirm {
+    padding-block: 0;
+    padding-inline: 1rem;
+}
+
+.dialog__info--confirm {
+    --light-clr: #5d768c2f;
```
Comment (ramalingaa) [inline pages/components/modal/modal.css:128]: try to define variable in the root

**Review by ramalingaa — APPROVED** [review]

File: pages/components/modal/modal.css (hunk 2 above)
Comment (manish591) [reply pages/components/modal/modal.css:128]: It was in the root. I'm overriding it for that parent block.

**Pull request merged**